### ISSUE TITLE
[➕ Feature]: Manually execute workflows on incidents

### ISSUE THREAD

**Issue by imorph** [keephq/keep#2197]
## Problem statement

It is cool that we are able to get bunch of different alerts to be correlated and batched into single incident automatically. But it will be even more useful if it would be possible to execute workflow on this incident. 

Exact user story may looks like this:
* bunch of alerts generated and batched into incident (lets say by a "server" field)
* user can create jira ticket about this incident by executing workflow
* workflow have access to all alerts within incident, so all of them listed in ticket (richer context available)
* incident have backlink to created ticket

Basically it is how it works right now with individual alerts but on incident level. For now if something happens with "server" we got separate alerts for "cpu", "disk", "network" etc and for each of them individually it is possible to create tickets but essentially this is single "Incident" and to have one jira ticket per one incident is great.

But may be I am missing something and this is already possible (we use 0.24.5)?

Thanks

**Comment by talboren** [comment]
Hey @imorph! Thanks for opening this issue!

This is already possible and was implemented by @VladimirFilonov. Happy to elaborate further on our [Slack](https://slack.keephq.dev): https://github.com/keephq/keep/pull/1861

**Comment by talboren** [comment]
I'm going to go ahead and close this issue, but if you feel like there's still place for this issue to exist, please re-open it 🙏🏼

**Comment by imorph** [comment]
@talboren thanks for quick reply!

As I understand that change, it is more about adding incident lifecycle events (create/update/delete) as automatic triggers for workflow execution. That is great addition, but I am talking about more manual (or user involving) use case when "operator" have lets say `15` incidents per day and only `5` of them are worthy creating ticket (execute "Create Ticket" workflow), for `6` of them there easy remediation exists (execute workflow "Easy Remediation") and rest `4` is not a problem so we can safely remove them.

**Comment by talboren** [comment]
@imorph thanks for the clarification. This absolutely deserves a ticket then :) we’ll get to it ASAP.

**Comment by VladimirFilonov** [comment]
Hello @imorph! I just merged the possibility of manually running workflows for incidents, which will be available in the next release.
